src/models/ContentItem.ts

```diff
@@ -0,0 +1,10 @@
+export class ContentItem {
+  name: string;
+  /**
+   * Field is excluded from the response when a locale is set
+   */
+  id: string;
+  localizedNames: string;
+  assetName: string;
+  assetPath: string;
```

Comment:
Why not include 
`This field is only included for maps and game modes. These values are used in the match response.` from the docs if including the other doc

Reply:
This has been included and placed above assetPath.

Reply:
`assetPath` I believe is optional as well based on the comment

Reply:
assetPath has been properly updated to be optional.